assets/js/googlesitekit/api/cache.js

```diff
@@ -22,12 +22,134 @@
 import { isEqual } from 'lodash';
 
 /**
- * Internal dependencies
+ * Prefix used for all Site Kit keys.
+ *
+ * Anything not using this key should not be touched by this library.
+ *
+ * @private
+ */
+export const STORAGE_KEY_PREFIX = 'googlesitekit_';
+
+const defaultOrder = [ 'localStorage', 'sessionStorage' ];
+let storageBackend;
+let storageOrder = [ ...defaultOrder	];
+/**
+ * Override the storage backend.
+ *
+ * Largely used for tests. Should not be used directly.
+ *
+ * @private
+ * @param {*} backend Backend to set for the cache.
+ * @return {void}
+ */
+export const setSelectedStorageBackend = ( backend ) => {
+	storageBackend = backend;
+};
+
+/**
+ * Override the priority of storage mechanisms.
+ *
+ * Largely used for tests. Implicitly resets the selected storage backend,
+ * causing `_getStorage` to re-run its checks for the best available
+ * storage backend.
+ *
+ * @private
+ * @param {Array} order Ordered array of storage backends to use.
+ * @return {void}
+ */
+export const setStorageOrder = ( order ) => {
+	storageOrder = [ ...order ];
+	setSelectedStorageBackend( undefined );
+};
+
+/**
+ * Reset the storage mechanism order.
+ *
+ * Largely used for tests. Implicitly resets the selected storage backend,
+ * causing `_getStorage` to re-run its checks for the best available
+ * storage backend.
+ *
+ * @private
+ * @return {void}
+ */
+export const resetDefaultStorageOrder = () => {
+	storageOrder = [ ...defaultOrder ];
+	setSelectedStorageBackend( undefined );
+};
+
+/**
+ * Detects whether browser storage is both supported and available.
+ *
+ * @private
+ * @param {string} type Browser storage to test. Should be one of `localStorage` or `sessionStorage`.
+ * @return {boolean} True if the given storage is available, false otherwise.
```

Comment:
These should use `bool`.

Reply:
We're using `{boolean}` everywhere else in the repo, and `Boolean` is the actual JS type, not `bool`.

If I try to rename it, ESLint complains (and then auto-fixes it for me):

![2020-03-09 20 45 45](https://user-images.githubusercontent.com/90871/76255958-02f48200-6247-11ea-94cd-44e99905deec.gif)

Reply:
I'll update that in our doc.